supplier_app/forms.py

```diff
@@ -61,29 +118,33 @@ class Meta:
         }
 
 
-class TaxPayerCreateForm(ModelForm):
+class TaxPayerCreateForm(BasePrefixCreateForm):
     prefix = 'taxpayer_form'
 
-    def __init__(self, user=None, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        # self.fields['company'].queryset = self.get_user_companies(user)
-
-    def get_user_companies(self, user):
-        return Company.objects.filter(companyuserpermission__user=user)
-
     class Meta:
         model = TaxPayerArgentina
         exclude = ['taxpayer_state', 'workday_id', 'company', 'country']
         widgets = {
             'name': forms.TextInput(attrs={'class': 'form-control'}),
-            'razon_social': forms.TextInput(attrs={'class': 'form-control'}),
+            'business_name': forms.TextInput(attrs={'class': 'form-control'}),
             'cuit': forms.TextInput(attrs={'class': 'form-control'}),
-            'forma_de_pago': forms.TextInput(attrs={'class': 'form-control'}),
-            'justificacion': forms.Textarea(attrs={'class': 'form-control', 'rows': '3'})
+            'payment_term': forms.TextInput(attrs={'class': 'form-control'}),
+            'comments': forms.Textarea(attrs={
+                'class': 'form-control',
+                'rows': '3'
+            }),
+            'AFIP_registration_file': forms.FileInput(attrs={
```

Comment:
I'd avoid capital letters on naming :)

Reply:
Ok I will consider in the future, in the next PR i will fix that